public/i18n/en/translation.json

```
@@ -1826,6 +1826,26 @@
       "password_suggestion_4": "At least one number (0-9)",
       "password_suggestion_5": "At least one special character (!{'@'}{'#'}^&_?-)",
       "current_password": "Current password"
+    },
+    "users": {
+      "title": "Users",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus lobortis interdum sollicitudin. Proin eget accumsan massa.",
```

```suggestion
      "description": "The users listed on this page are granted full control over all firewall units. Exclusive permissions to create, modify, or delete user accounts are reserved solely for the administrator."",
```

What do you think @andre8244 ?